### ISSUE TITLE
Same point cloud input - different numeric results

### ISSUE THREAD

**Issue by RanBenIzhak** [HuguesTHOMAS/KPConv#44]
Hi, @HuguesTHOMAS 

First of all - great work, and extremely well documented! 

After training your model using the ModelNet40 config, I am doing some testing on new dataset (the remarks and readme were very helpful).
I receive 2 different outputs for the same point cloud as input to the network (using ModelNet40 config, deformable conv, output is the layer before the classification head).
The differences are significant in values (up to 0.25 of the vectors Norm)

run command is
sess.run(ops, {model.dropout_prob: 1.0})

Is it a normal behavior of the model? is any part of the computation is non-deterministic?

Thanks,
Ran

**Comment by RanBenIzhak** [comment]
Edit:
Found it might be a bug in another place (my modifications).
closing, will re-open after further investigation if needed

**Comment by HuguesTHOMAS** [comment]
HI @RanBenIzhak,

The only non deterministic part in the computations is the data augmentation process. You can disable it by changing the related parameters in the config after loading it and before starting the model. if you have already done that, the result should be the same for two identical input point clouds.

Best,
Hugues
